tasks/check_for_ca.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Puppet Task Name: check_for_ca
+#
+# Explected possible inputs:
+#   $PT_master_replica_host
+#   $PT_puppetdb_database_host
+#   $PT_puppetdb_database_replica_host
+#   $PT_compiler_hosts
+#
+
+# A function to check for and remove signed certs for the infra
+function check_for_cert()
```

Comment:
The function name implies that a check is only performed, yet a removal is actually done.  Consider renaming to check_n_remove_cert